sdk/storage/azblob/appendblob/models.go

```diff
@@ -73,8 +74,12 @@ func (o *CreateOptions) format() (*generated.AppendBlobClientCreateOptions, *gen
 
 // AppendBlockOptions contains the optional parameters for the Client.AppendBlock method.
 type AppendBlockOptions struct {
-	// Specify the transactional crc64 for the body, to be validated by the service.
-	TransactionalContentCRC64 []byte
+	// Let the SDK hash for you (providing the hash type(s) specified voids their relevance in this option).
+	// nil = None default
+	TransactionalValidationOption blob.TransferValidationType
```

Comment:
What do you think about shortening this to `TransactionalValidation`?

Reply:
Sounds like a good change to make